src/arrays/programs_involving_multidimensional_data/easy/q3/Identities.java

```diff
@@ -0,0 +1,48 @@
+package arrays.programs_involving_multidimensional_data.easy.q3;
+
+/*
+Write a Java program that generates a 2D array of size 3 by 3 made up of random integers that are either 0 or 1, and
```

Comment:
Change to:
"Write a Java program that randomly generates a 2D integer array of size 3 by 3 where each element is either 0 or 1, and prints it. Pass that array to a method called checkMatrix which prints out true if the array is an identity matrix and false otherwise. Recall that an identity matrix is a square array where every element is 0 except the elements in the top left to the bottom right diagonal which are 1's."